examples/web/postcss.config.js

```diff
@@ -1,6 +1,9 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
ESLint cannot resolve this dependency because example has its own package.json.

Reply:
Would you please drop it as a comment right into the code?

Reply:
Ok, added.